include/zephyr/dt-bindings/pinctrl/gecko-pinctrl-s1.h

```diff
@@ -43,6 +44,16 @@
 /** Mask for the pin field. */
 #define GECKO_LOC_MSK 0xFFU
 
+/** Position of low power field. */
+#define GECKO_LP_POS 16U
+/** Mask for the low power field. */
+#define GECKO_LP_MSK 0x1U
+
+/** Input. */
+#define	GECKO_LP_DISABLE 0U
+/** Output. */
+#define GECKO_LP_ENABLE 1U
```

Comment:
I think it does not make sense to define symbols that are finally just another name for boolean.

Reply:
you are right: it is just more "futurproof" if more bits are required for low-power feature: I let you decide.
I followed the nordic implementation (pinctrl-nrf)

Reply:
YAGNI. So, I suggest to drop them.